### PR TITLE
BZ#1446667-Added rsync command to pull files from the pod

### PR DESCRIPTION
**Pull request by gaurav-nelson**
- Added rsync command to pull files from the pod
- Linked it to the suggested example

### CODE REVIEW

**Review by sferich888 — APPROVED** [review]
LGTM

**Comment by gaurav-nelson** [comment]
@bfallonf PTAL

File: admin_guide/backup_restore.adoc [hunk 1]
```diff
@@ -823,7 +823,7 @@ accounts).
 In many cases, application data can be backed up using the `oc rsync` command,
 assuming `rsync` is installed within the container image. The Red Hat *rhel7*
 base image does contain `rsync`. Therefore, all images that are based on *rhel7*
-contain it as well.
+contain it as well. See xref:../cli_reference/basic_cli_operations.adoc#cli-operations-rsync[Troubleshooting and Debugging CLI Operations - rsync]
```
Comment (bfallonf) [inline admin_guide/backup_restore.adoc]: Full stop after this?

File: cli_reference/basic_cli_operations.adoc [hunk 2]
```diff
@@ -589,14 +589,21 @@ Open a remote shell session to a container:
 ----
 $ oc rsh <pod>
 ----
-
+[[cli-operations-rsync]]
 === rsync
-Copy contents of local directory to a directory in an already-running pod
-container. It will default to the first container if none is specified.
+Copy the contents to or from a directory in an already-running pod
+container. If you do not specify a container, it defaults to the first container.
```
Comment (bfallonf) [inline cli_reference/basic_cli_operations.adoc]: I'm not too sure what "first container" means? Can this be clarified?

**Comment by bfallonf** [comment]
@gaurav-nelson Just a couple comments. LG otherwise.

**Comment by gaurav-nelson** [comment]
@bfallonf Thanks for reviewing :white_check_mark: 

Added missing full stop, and updated the text to say " first container in the pod."

**Comment by vikram-redhat** [comment]
@gaurav-nelson - does this require a rev history?

**Comment by gaurav-nelson** [comment]
Thanks @vikram-redhat - Updated labels and milestone.
No revision history needed.

**Pull request merged**